/*const myPromise = new Promise((resolve, reject) => {
  setTimeout(() => {
    const user = { name: "user" };
    resolve(user);
  }, 3000);
});

myPromise
  .then((res) => console.log("data-", res))
  .catch((err) => console.log("error-", err));*/
